en/appbuilder/generate-app/upload-application-to-github.md

```diff
@@ -0,0 +1,44 @@
+---
+# Upload Application To Github
+
+> [!NOTE]
+><b>Once ready with you design in Indigo.Design App Builder you can upload it to your private or Github repository. 
+
+
+### In this article:
+* <a href="#intro">Introduction of the feature</a>
+* <a href="#uploading-an-application-to-github">Uploading an application to GitHub</a>
+
+## Intro
+One of the great Indigo.Design App Builder features is actually the code generation service, so once the app design is ready and the code reviewed, users can get the final application code. However, what makes the experience even greater is the fact that the app code can be easily uploaded to a GitHub repository directly from inside the App Builder. GitHub is a favorite tool for thousands of developers, so this App Builder feature allows users to benefit from all GitHub features, such as assigning reviewers, tracking changes, commenting and many more. By delivering this feature, the Indigo.Design product team adds one more piece to the full design-to-code collaborative story, but also integrates the App Builder platform with a well-known service that engineers are already used to. 
+
+## Uploading an application to GitHub
+After finalizing the design of your application and previewing its code, it comes the time for generating the full app code repository. In order to do that, simply go to the Generate app button and then connect your GitHub account to the App Builder. You can also download you application files from the button menu.
+
+
+<img class="responsive-img" src="../../images/connect-to-github.png" srcset="../../images/connect-to-github-@2x.png 2x" />
+<p style="text-align:center;">Connect Indigo.Design App Builder to GitHub</p>
+
+Once you log in with your GitHub account, you should authorize Indigo.Design to access it in order to pick up the repository where the app code will be stored. There, you can select between creating a new repository or adding to an existing one. When creating a new repository you can specify the name of the app, add description and choose between public or private repo. Once ready, the app files are being uploaded to GitHub, creating a new GitHub repository.
+
+<img class="responsive-img" src="../../images/pick-repository-publish-to-github.png" srcset="../../images/pick-repository-publish-to-github-@2x.png 2x" />
+<p style="text-align:center;">Publish application to an existing repo in GitHub</p>
+
+<img class="responsive-img" src="../../images/create-new-repo-publish-to-github.png" srcset="../../images/create-new-repo-publish-to-github-@2x.png 2x" />
+<p style="text-align:center;">Create a new repository to GitHub</p>
+
+When the upload process is done, users can apply changes to the GitHub repo from inside the App Builder or click to "View it on GitHub" and open the browser to check the updated files in GitHub.
```

Comment:
```suggestion
When the upload process is done, users can apply changes to the GitHub repo from inside the App Builder or click to "View it on GitHub" and see the updated files in GitHub.
```
They are already in the browser using app builder so I think you can skip that part